modules/solid_mechanics/src/materials/SolidModel.C

```diff
@@ -989,7 +994,10 @@ SolidModel::crackingStrainDirections()
   {
     // Adjust the elasticity matrix for cracking.  This must be used by the
     // constitutive law.
-    _local_elasticity_tensor->adjustForCracking( _crack_flags_local );
+    if (_compute_method >= 2)
+      _local_elasticity_tensor->adjustForCracking( _crack_flags_local, true);
+    else
+      _local_elasticity_tensor->adjustForCracking( _crack_flags_local, false);
```

Comment:
Ugh, that's not quite that easy, as the multipliers would not be available in the new method. Never mind then. Alternative suggestion would be to pass in a custom `enum` (`WITH_SHEAR_RETENTION`, `WITHOUT_SHEAR_RETENTION`) to make the call more expressive.